jolt-core/src/test/java/com/bazaarvoice/jolt/modifier/function/MathTest.java

```diff
@@ -227,8 +232,28 @@
 
 
         testCases.add( new Object[] { "toInteger-combo-string-array", TO_INTEGER, Arrays.asList( "-1", 2, -3L, 4.0 ), Optional.of( Arrays.asList( -1, 2, -3, 4 ) ) } );
-        testCases.add( new Object[] { "toLong-combo-int-array", TO_LONG, Arrays.asList( "-1", 2, -3L, 4.0 ), Optional.of( Arrays.asList( -1L, 2L, -3L, 4L ) ) } );
-        testCases.add( new Object[] { "toDouble-combo-long-array", TO_DOUBLE, Arrays.asList( "-1", 2, -3L, 4.0 ), Optional.of( Arrays.asList( -1.0, 2.0, -3.0, 4.0 ) ) } );
+        testCases.add( new Object[] { "toLong-combo-int-array",       TO_LONG,    Arrays.asList( "-1", 2, -3L, 4.0 ), Optional.of( Arrays.asList( -1L, 2L, -3L, 4L ) ) } );
+        testCases.add( new Object[] { "toDouble-combo-long-array",    TO_DOUBLE,  Arrays.asList( "-1", 2, -3L, 4.0 ), Optional.of( Arrays.asList( -1.0, 2.0, -3.0, 4.0 ) ) } );
+
+        testCases.add( new Object[] { "intsum-combo-string-array",    INT_SUM_OF,  Arrays.asList(1, 2.0, "random", 0), Optional.of(3)});
+        testCases.add( new Object[] { "intsum-single-value",          INT_SUM_OF,  2,                                  Optional.empty()});
+        testCases.add( new Object[] { "intsum-combo-intstring-array", INT_SUM_OF,  Arrays.asList(1L, 2, "-3.0", 0), Optional.of(0)});
+
+        testCases.add( new Object[] { "doublesum-combo-string-array",    DOUBLE_SUM_OF, Arrays.asList(1, 2.0, "random", 0), Optional.of(3.0)});
+        testCases.add( new Object[] { "doublesum-single-value",          DOUBLE_SUM_OF, 2,                                  Optional.empty()});
+        testCases.add( new Object[] { "doublesum-combo-intstring-array", DOUBLE_SUM_OF, Arrays.asList(1L, 2, "-3.0", 0),    Optional.of(0.0)});
+
+        testCases.add( new Object[] { "longsum-combo-string-array",      LONG_SUM_OF,   Arrays.asList(1, 2.0, "random", 0), Optional.of(3L)});
+        testCases.add( new Object[] { "longsum-single-value",            LONG_SUM_OF,   2, Optional.empty()});
+        testCases.add( new Object[] { "longsum-combo-intstring-array",   LONG_SUM_OF,   Arrays.asList(1L, 2, "-3.0", 0),    Optional.of(0L)});
+
+        testCases.add( new Object[] { "div-combo-array",          DIV_OF, Arrays.asList(10L, 5.0, 2), Optional.of(2.0)});
```

Comment:
comment that the 3rd param in the list is there just to prove that extra params don't break things.